jakeruss-capecoral.md

```diff
@@ -0,0 +1,8 @@
+# Cape Coral, FL
+
+### Restaurants
+
+### Tourist Attractions
+
+### Local Attractions
+- Cayo Costa State Park
```

Comment:
It is an island in the gulf, take the ferry over to the beach, or bring camping gear!